api/api_test.go

```diff
@@ -157,11 +155,74 @@ func TestValidDeploymentRequestAndManifestCreateResources(t *testing.T) {
 		Application: appName,
 		Version:     version,
 		Environment: environment,
+		Username:    "user",
+		Password:    "password",
```

Comment:
Are these really fasit-related? Maybe name them `fasitUsername` and `fastPassword`.

Reply:
that will defeat the purpose of the test